<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtFishClientClass</class>
 <widget class="QMainWindow" name="QtFishClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtFishClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tcpTextBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>621</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Acceleration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send_0x67</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>911</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>430</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>390</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>470</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>550</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>target</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>510</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>550</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>590</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPosz_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>570</y>
      <width>239</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPosx_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>517</y>
      <width>239</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPosy_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>544</y>
      <width>239</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>340</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startVideoBtn">
       <property name="text">
        <string>show video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartCamera">
       <property name="text">
        <string>Start Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStopCamera">
       <property name="text">
        <string>Stop Camera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCalibrate">
       <property name="text">
        <string>Calibrate Camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPosx">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>391</y>
      <width>149</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPosy">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>421</y>
      <width>149</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPosz">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>451</y>
      <width>149</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserAccelx">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>171</y>
      <width>159</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserAccely">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>202</y>
      <width>159</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserAccelz">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>234</y>
      <width>159</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserYaw">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>328</y>
      <width>159</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPitch">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>265</y>
      <width>159</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserRoll">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>297</y>
      <width>159</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>171</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>202</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>234</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>328</y>
      <width>18</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>yaw</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>297</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>265</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pitch</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserDirection">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>391</y>
      <width>159</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserSpeed">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>360</y>
      <width>159</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>360</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>391</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>direction</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtFishClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked()</signal>
   <receiver>QtFishClientClass</receiver>
   <slot>OnBtnConnectClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSend</sender>
   <signal>clicked()</signal>
   <receiver>QtFishClientClass</receiver>
   <slot>OnBtnSendClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSend_2</sender>
   <signal>clicked()</signal>
   <receiver>QtFishClientClass</receiver>
   <slot>OnBtnSend_0x67Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartCamera</sender>
   <signal>clicked()</signal>
   <receiver>QtFishClientClass</receiver>
   <slot>StartCameraThread()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>865</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>943</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnAdd1Change(int)</slot>
  <slot>OnAdd1Change()</slot>
  <slot>OnBtnConnectClick()</slot>
  <slot>OnBtnSendClick()</slot>
  <slot>OnBtnSend_0x67Click()</slot>
  <slot>ShowCameraFrame()</slot>
  <slot>StartCameraThread()</slot>
 </slots>
</ui>
